v2/tools/generate-docs.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus"
+	_ "github.com/datadog/stratus-red-team/v2/pkg/stratus/loader"
+)
+
+func main() {
```

Comment:
This is the entry point to generate docs. This functions call `GenerateTechDocs` to generate all `md` files and `GenerateYAML` to generate `docs/index.yaml`.